c2corg_ui/static/js/utils.js

```diff
@@ -123,6 +123,30 @@ app.utils.formatDate = function(date) {
 
 
 /**
+ * check that the start date if before end date.
+ * Returns an error message if there is an error with dates.
+ * @export
+ * @param {Object} outing
+ * @return {(undefined | string)}
+ */
+app.utils.checkDates = function(outing) {
```

Comment:
I think it would be better.
"mieux vaut prévenir que guérir" "an ounce of prevention is worth a pound of cure"